.env.sample

```diff
@@ -1,4 +1,4 @@
-WORDPRESS_TITLE=Skela Site
+WORDPRESS_TITLE="Skela Site"
 WORDPRESS_THEME_NAME=skela
```

Comment:
Ugh, this is due to sharing `.env` file across application config and Docker. I'll find a way to make this work in a separate PR so we can stop having site names with embedded quotes...